Edit search bar for docs (#2134)
meilisearch/documentation: learn/cookbooks/search_bar_for_docs.md

You might have noticed the search bar in this documentation.

> ![Meilisearch docs search bar updating results for 'faq'](https://i.ritzastatic.com/images/d684286ffb7a420c87da673a1a2ae107/meilisearch-bar-example.gif)

Reviewer: ```suggestion
![Meilisearch docs search bar updating results for 'faq'](/tuto-searchbar-for-docs/vuepress-searchbar-demo.gif)
```

```bash
curl -L https://install.meilisearch.com | sh

> ./meilisearch --master-key=myMasterKey

Reviewer: ```suggestion
./meilisearch --master-key=MASTER_KEY
```

docker run -t --rm \
  --network=host \
  -e MEILISEARCH_HOST_URL='http://localhost:7700' \

>   -e MEILISEARCH_API_KEY='myMasterKey' \

Reviewer: ```suggestion
  -e MEILISEARCH_API_KEY='MASTER_KEY' \
```

</html>
```

> The `hostUrl` and the `apiKey` fields are the credentials of the Meilisearch instance. Following on from this tutorial, they are respectively `http://localhost:7700` and `myMasterKey`.

Reviewer: ```suggestion
The `hostUrl` and the `apiKey` fields are the credentials of the Meilisearch instance. Following on from this tutorial, they are respectively `http://localhost:7700` and `MASTER_KEY`.
```
